kjourney/src/app/app.component.css

```diff
@@ -2,14 +2,16 @@
   box-sizing: border-box;
 }
 
-body {
+/* body {
   background-image: linear-gradient(to bottom right, #61ab3f, white);
-  background-attachment: fixed;
-  background-size: cover;
-  overflow: hidden;
-  height: 100vh;
+  height: 100%;
   width: 100%;
-  font-family: sans-serif;
+} */
```

Comment:
why commented out? can this just be deleted?

Reply:
sorry who are you?

Many thanks,
Gordon